### PR TITLE
City view first dev implementation

### PR DESCRIPTION
**Pull request by luisferreira32**
Do some modifications on the Makefiles for easier development experience. Do the first micro service implementation with the information necessary to show the city view (not yet build it).

### CODE REVIEW

File: databases/gamedb/init.sql [hunk 1]
```diff
@@ -77,8 +77,6 @@ CREATE TABLE city(
     b_mutant_lab_level INT,
     b_temple_level INT,
     income_tax FLOAT,
-    fishboats_amount INT,
```
Comment (luisferreira32) [inline databases/gamedb/init.sql:80]: this was duplicated in the units with u_ prefix

File: databases/gamedb/Makefile [hunk 2]
```diff
@@ -2,10 +2,10 @@ build:
 	docker build -t stickianempire/gamedb:0.1 .
 
 dev-run: build
-	docker run -e POSTGRES_PASSWORD=postgres -e POSTGRES_USER=postgres -d -p 5433:5432 stickianempire/gamedb:0.1
+	docker run --name dev-postgres -e POSTGRES_PASSWORD=postgres -e POSTGRES_USER=postgres -d -p 5432:5432 stickianempire/gamedb:0.1
```
Comment (luisferreira32) [inline databases/gamedb/Makefile:5]: fixed here

File: databases/gamedb/Makefile [hunk 3]
```diff
@@ -2,10 +2,10 @@ build:
...
 	@echo "Your DB is running with localhost:5432 and username/password: postgres/postgres"
-	@echo "If you wish to exec into it use \"docker exec -it <contained-id> bash\""
+	@echo "If you wish to exec into it use \"docker exec -it dev-postgres bash\""
 
 
 dev-clean:
-	docker ps | grep "stickianempire" | awk '{system("docker kill "$$1)}'
+	docker ps -a| grep "stickianempire" | awk '{print $$1}' | xargs -d '\n' bash -c 'for arg do docker stop $$arg; docker rm $$arg; done' _
```
Comment (luisferreira32) [inline databases/gamedb/Makefile:11]: stackoverflow magic commands

File: services/main.go [hunk 4]
```diff
@@ -0,0 +1 @@
+package main
```
Comment (luisferreira32) [inline services/main.go:1]: can have an implementation in the future when we have multiple game micro services and we want to compile it all in one binary :)

**Review by antoniomf97 — APPROVED** [review]
Seems all good, just take

**Pull request merged**